crates/bevy_app/src/lib.rs

```diff
@@ -7,6 +7,8 @@
 
 //! This crate is about everything concerning the highest-level, application layer of a Bevy app.
 
+extern crate alloc;
```

Comment:
I don't think alloc is used here. Can this be removed?

Reply:
Unfortunately not, `bevy_app` uses `Cow` and `Rc` from `alloc`.